internal/cmd/db_create.go

```diff
@@ -70,6 +89,14 @@ var createCmd = &cobra.Command{
 			Password: res.Password,
 		}
 
+		if dbFile != nil {
```

Comment:
I wonder if it makes a little more sense to have all the logic in one place after the database has been created?

```go
if dbFromFile != "" {
    dbFile, err := getDbFile(dbFromFile)
    if err != nil {
        return err
    }
    defer dbFile.Close()
    err := client.Databases.Seed(name, dbFile)
    if err != nil {
        client.Databases.Delete(name)
        return fmt.Errorf("could not create database %s: %w", name, err)
    }
}
```

Reply:
Maybe it is better to check that the file exists first before creating the database though, I can see why you went that route instead.

Reply:
yeah, mostly we want to make sure that system-related errors don't even trigger a database creation event.